nengo/spa/vocab.py

```diff
@@ -480,15 +485,63 @@ def create_subset(self, keys):
         return subset
 
 
-class VocabularyParam(nengo.params.Parameter):
-    """Can be a Vocabulary."""
+class VocabularyMap(Mapping):
```

Comment:
It would be nice to have a quick one-line docstring about the general intended purpose of this object.